packages/extensions/cloud/package.json

```diff
@@ -1,19 +1,19 @@
 {
-  "name": "@n8n/n8n-extension-insights",
+  "name": "@n8n/n8n-extension-cloud",
```

Comment:
Can we simplify naming? e.g. `@n8n/cloud` as it's clear it's an extension from the dir name. Also ideally `module` in the dirname instead of `extension` as this is internal.